Assets/style.css

```diff
@@ -23,7 +23,7 @@
   }
   
   .ui.icon.input {
-      padding-top: 20px
+      padding-top: 20px;
```

Comment:
You can delete the css for ui.icon.input because for some reason it puts the search icon off-center.